board/templates/home.html

```diff
@@ -1,472 +1,1015 @@
-                  }});
                 
+            }
+            i+=1
+          }
+          // $.each(workspaces,function(_,workspace){
+          //     addWorkspaceToUI(workspace)
+          // });
+      }
+      function getModelsDiv(models, wid){
+          models_div= ""
+          var models = eval(models);
+          console.log("models:",models)
+          for(var model_id in models){
+              if (models.hasOwnProperty(model_id)){
+                  var model_details=models[model_id]; //[status,port,file_location]
+                  model_status = model_details[0]
+                  model_port = model_details[1]
+                  model_file_location = "user/workspace/model/download?mid="+model_id
+                  if (model_status == "TRAINED"){
+                      // show download and deploy options
+                      models_div += ('<div class="col-md-12 col-sm-12 col-xs-12">\
+                    <div class="x_panel">\
+                      <div class="x_title">\
+                        <h2>Model: '+model_id+' <small>model description/tags</small></h2>\
+                        <ul class="nav navbar-right panel_toolbox">\
+                          <li>\
+                              <span class="trained label label-success" pwid="'+wid+'" style="color:#FFF;vertical-align: bottom;">Training Completed</span>\
+                          </li>\
+                          <li><a class="collapse-link"><i class="fa fa-chevron-up"></i></a>\
+                          </li>\
+                          <li class="dropdown">\
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false"><i class="fa fa-gear"></i></a>\
+                            <ul class="dropdown-menu" role="menu">\
+                              <li><a href="#">Get Sharing link</a>\
+                              </li>\
+                              <li><a href="#">Notify on completion <i class="fa fa-check"></i></a>\
+                              </li>\
+                            </ul>\
+                          </li>\
+                          <li><a class="close-link"><i class="deleteModel fa fa-trash"></i></a>\
+                          </li>\
+                        </ul>\
+                        <div class="clearfix"></div>\
+                      </div>\
+                      <div class="x_content">\
+                        <div class="col-md-12 col-sm-12 col-xs-12" style="border:0px solid #e5e5e5;">\
+                            <button type="button" class="btn btn-primary">Download model</button>\
+                        </div>\
+                    </div>\
+                  </div>\
+                </div>')
+                      // models_div += ('<div mid="'+model_id+'" class="model-div" mstatus="'+model_status+'">Model:'+model_id+'<a href="'+model_file_location+'"><i class="fa fa-download"></i> Download</a> <i class="fa fa-trash deleteModel"></i></div>')
```

Comment:
Nit: do we want to keep this commented code? (and similar other commented codes in this file)

Reply:
yes, work related to periodic model status checks and deletion is pending, i need for reference, this can be safely deleted once that portion is complete.

Reply:
most of the commented code is kept to remember UI functions usage from previous UI, some migration part is left. It would be safe to clean up after that migration is complete along with some basic testing (which will reveal UI bugs in upcoming days.).